docs/getting-started-guides/minikube.md

```diff
@@ -55,6 +55,35 @@ Minikube requires one of the following:
 
 See the [latest Minikube release](https://github.com/kubernetes/minikube/releases) for installation instructions.
 
+### Download `kubectl`
+
+You will need to download the kubectl client binary for `${K8S_VERSION}` (in this example: `{{page.version}}.0`)
+to run commands against the cluster.
+
+Downloads:
+
+ - `linux/amd64`: http://storage.googleapis.com/kubernetes-release/release/{{page.version}}.0/bin/linux/amd64/kubectl
+ - `linux/386`: http://storage.googleapis.com/kubernetes-release/release/{{page.version}}.0/bin/linux/386/kubectl
+ - `linux/arm`: http://storage.googleapis.com/kubernetes-release/release/{{page.version}}.0/bin/linux/arm/kubectl
+ - `linux/arm64`: http://storage.googleapis.com/kubernetes-release/release/{{page.version}}.0/bin/linux/arm64/kubectl
+ - `linux/ppc64le`: http://storage.googleapis.com/kubernetes-release/release/{{page.version}}.0/bin/linux/ppc64le/kubectl
+ - `OS X/amd64`: http://storage.googleapis.com/kubernetes-release/release/{{page.version}}.0/bin/darwin/amd64/kubectl
+ - `OS X/386`: http://storage.googleapis.com/kubernetes-release/release/{{page.version}}.0/bin/darwin/386/kubectl
+ - `windows/amd64`: http://storage.googleapis.com/kubernetes-release/release/{{page.version}}.0/bin/windows/amd64/kubectl.exe
+ - `windows/386`: http://storage.googleapis.com/kubernetes-release/release/{{page.version}}.0/bin/windows/386/kubectl.exe
+
+The generic download path is:
+```
+http://storage.googleapis.com/kubernetes-release/release/${K8S_VERSION}/bin/${GOOS}/${GOARCH}/${K8S_BINARY}
+```
+
+An example install with `linux/amd64`:
+
+```
+curl -sSL "http://storage.googleapis.com/kubernetes-release/release/{{page.version}}.0/bin/linux/amd64/kubectl" > /usr/bin/kubectl
```

Comment:
Replace {{page.version}} with, for example, 1.2 and it works great.

http://storage.googleapis.com/kubernetes-release/release/v1.2.0/bin/linux/amd64/kubectl